examples/remarketing/upload_offline_conversion.py

```diff
@@ -0,0 +1,92 @@
+         conversion_value):
+    """Creates a click conversion with a default currency of USD."""
+
+    click_conversion = client.get_type('ClickConversion', version='v2')
+    conversion_action_service = client.get_service('ConversionActionService',
+                                                   version='v2')
+    click_conversion.conversion_action.value = (
+        conversion_action_service.conversion_action_path(
+            customer_id, conversion_action_id)
+    )
+    click_conversion.gclid.value = gcl_id
+    click_conversion.conversion_value.value = float(conversion_value)
+    click_conversion.conversion_date_time.value = conversion_time
+    click_conversion.currency_code.value = 'USD'
+
+    conversion_upload_service = client.get_service('ConversionUploadService',
+                                                   version='v2')
+
+    try:
+        conversion_upload_response = (
+            conversion_upload_service.upload_click_conversions(customer_id,
+                [click_conversion], partial_failure=True)
+        )
+        uploaded_click_conversion = (conversion_upload_response.results[0])
+        print(f'Uploaded conversion that occurred at '
+              f'"{uploaded_click_conversion.conversion_date_time.value}" '
+              f'from Google Click ID "{uploaded_click_conversion.gclid.value}"'
+              f' to "{uploaded_click_conversion.conversion_action.value}"')
+
+    except GoogleAdsException as ex:
+        print('Request with ID "%s" failed with status "%s" and includes the '
+              'following errors:' % (ex.request_id, ex.error.code().name))
+        for error in ex.failure.errors:
+            print('\tError with message "%s".' % error.message)
+            if error.location:
+                for field_path_element in error.location.field_path_elements:
+                    print('\t\tOn field: %s' % field_path_element.field_name)
+        sys.exit(1)
+
+
+if __name__ == '__main__':
+    # GoogleAdsClient will read the google-ads.yaml configuration file in the
+    # home directory if none is specified.
+    google_ads_client = GoogleAdsClient.load_from_storage()
+
+    parser = argparse.ArgumentParser(
+        description='Upload an offline conversion.')
+    # The following argument(s) should be provided to run the example.
+    parser.add_argument('-c', '--customer_id', type=str,
+                        required=True, help='The Google Ads customer ID.')
+    parser.add_argument('-a', '--conversion_action_id', type=str,
+                        required=True, help='The conversion action ID.')
+    parser.add_argument('-g', '--gcl_id', type=str,
+                        required=True, help='The Google Click Identifier ID.')
+    parser.add_argument('-t', '--conversion_time', type=str,
+                        required=True, help='The conversion time.')
+    parser.add_argument('-v', '--conversion_value', type=str,
+                        required=True, help='The conversion value.')
```

Comment:
The helpers miss information compare to the golden (e.g. the format and constraint of "conversion_time")

Reply:
Added.